src/app/commands/cat.command.ts

```diff
@@ -46,8 +46,8 @@ const command: Command = {
     let searchCom = '';
 
     const breedIn = interaction.options.getString('breed')?.toLowerCase() ?? 'random';
-    // checks if their was a bread was a breed, then if that breed is recognized
-    const breedEntry = ALL_BREEDS.find((breed) => breed.name === breedIn);
+    // checks if their was a breed was a breed, then if that breed is recognized
+    const breedEntry = ALL_BREEDS.find((breed) => breed.name.toLowerCase() === breedIn);
```

Comment:
there is a lower-cased breedNames list at the top of the file

Reply:
That has list added to it though, which isn't a breed,  unless you want me to check against list specifically.

Reply:
Oh wait, disregards you're talking about query

Reply:
the `list` case is handled on 36

Reply:
Can't use breedNames or query.  they are arrays, we need the full set since we need the ID for the breed.